runtime/browser/browser-vr-demo/browser-vr-demo.js

```diff
@@ -121,23 +121,9 @@ suggest(stage++);
 
 let chooseSuggestion = index => {
   document.querySelector('[particle-container]').textContent = '';
-  arc = cloneArc(arc); //arc.clone(); //prepareExtensionArc();
```

Comment:
Scott, I tried to run the VR demo, but it didn't work (only saw 3 tiny light-grey squares on the top of the page) - do i need to configure/install anything?